plugins/outputs/iotdb/sample.conf

```
@@ -0,0 +1,55 @@
+# Save metrics to an IoTDB Database
+[[outputs.iotdb]]
+  ## Configuration of IoTDB server
+  ## host is the IP address of the IoTDB server
+  ## port is a string descirbing the port of the IoTDB service, default port is 6667
+  host = "127.0.0.1"
+  port = "6667"
+
+  ## Configuration of authentication
+  ## The defualt user is 'root', and the defualt password is also 'root'
+  user = "root"
+  password = "root"
+
+  ## Configuration of session
+  ## - timeout is in milliseconds(ms). This is the timeout for calling 'Session.Open'
+  ##   The value type of 'timeout' should be int. 0 means no timeout. 
+  ##   Negative values will be treated as 0.
+  timeout = 5000
+
+  ## Configuration of type conversion for 64-bits unsigned int
+  ## IoTDB DO NOT support unsigned int (version 13.x). 
+  ## For uint32, this plugin will convert it into int64.
+  ## But if the specific type of an unsigned int is uint64, overflow may take place. 
+  ## So user should choose an available option of converting uint64 below.
+  ## 
+  ## This plugin supports 3 different available conversions of UInt64: 
+  ## - "ForceToInt64": no mater what the value it is, force covert it into int64.
+  ## - "ToInt64"(default): if an unsigned int is less than MAXINT64, covert it into int64; 
+  ##                       else save MAXINT64 instaed. math.MaxInt64 = 9223372036854775807
+  ## - "Text": no matter what value it is, convert it into a string, which is called TEXT in IoTDB.
+  convertUint64To = "ToInt64"
```

In TOML we use snake-case. Also here I think you should make the text less technical as not all users (should) know the implementation details...
```suggestion
  ## Configuration of type conversion for 64-bit unsigned int
  ## IoTDB currently DOES NOT support unsigned integers (version 13.x). 
  ## 32-bit unsigned integers are safely converted into 64-bit signed integers by the plugin,
  ## however, this is not true for 64-bit values in general as overflows may occur.
  ## The following option allows to specify the handling of 64-bit unsigned integers.
  ## Available options are:
  ##   - "int64"       --  convert to 64-bit signed integers and accept overflows
  ##   - "int64 clip"  --  convert to 64-bit signed integers and clip the values on overflow to 9,223,372,036,854,775,807
  ##   - "string"      --  convert to the string representation of the value
  # uint64_conversion = "int64 clip"
```

---

This is cleaner and more beautiful! Thanks for suggestion.